### ISSUE TITLE
Keep entries persistent as favorites

### ISSUE THREAD

**Issue by pyriand3r** [Keruspe/GPaste#217]
Hi,
First things first: I love gpaste, great tool! Can't think of my daily work without it any more. Would be horrible.

But i had the idea of beeing able to use gpaste not only as a clipboard history but also as a tool to have often used comands in easy reach. Kind of a snippet manager.
 I'm thinking about the possiblity to mark history entries as "favorites" moving them out of the clipboard history into a special "favorites" list, which would be persistent. It could be accessible through a second tab in the gui. Maybe sortable to be able to set up a persistent order as well.

**Comment by Keruspe** [comment]
Hi,

I love this idea and I think I know how it could be implemented.
I'll give it a try as soon as I can (hopefully in a couple of weeks)

**Comment by mat-m** [comment]
+1

**Comment by sbhouse** [comment]
+1

This would really help me at work!

**Comment by cr22rc** [comment]
+1 It's been a while since asked.  Any idea of an ETA? 
I'd like to mark entries as favorites that stay permanent. Hotkey that can cycle through these showing them in something like a pop-up.
Thx!

**Comment by breier** [comment]
+1 I think I may be able to help with that.
I'll take a look and ask some questions if @Keruspe don't mind ;)

**Comment by mbousq** [comment]
Hi @breier , any luck so far ?

**Comment by breier** [comment]
> Hi @breier , any luck so far ?

Hi @mbousq , sorry, I ended up moving jobs during that time and I completely forgot about it, hehehe...

Thanks for bringing that up again. I can definitely take a look this month.

**Comment by fc-asad** [comment]
Looking forward for this feature.

**Comment by KOT040188** [comment]
It's been 7 years!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Keruspe** [comment]
Yes, 7 years. I now have 4 kids, 3 more than 7 years ago. I still earn 0€ from maintaining GPaste but I still do it anyways.
I don't need this feature so it's very low in my todo list.
I'm not a company not backed by any for this 🙂

**Comment by VPashkov** [comment]
Thank you for your hard work! We really appreciate it.

**Comment by KOT040188** [comment]
Nobody forced the author to do this. He himself promised to do this. Empty promises...

**Comment by Keruspe** [comment]
It's now in master and will be part of the 51 release
